docker-app/qfieldcloud/notifs/templates/notifs/notification_email_body.html

```diff
@@ -0,0 +1,13 @@
+<p>Here is what happened recently on QFieldCloud.</p>
+
+<ul>
+    {% for notif in notifs %}
+    <li>{{notif}}</li>
+    {% endfor %}
+</ul>
+
+<hr/>
+
+<p>
+    <a href="https://{{hostname}}/settings/{{username}}/notifications_settings">Manage your notifications settings</a>
```

Comment:
Probably passing the URL from outside would easier to manage. Imagine `settings` renamed to `whatever` and you need to change at several places.